### PR TITLE
Correcting how to configure CNAME for a subdomain

### PR DESCRIPTION
**Pull request by theandyyeh**
### Why:

Usually, to configure a subdomain, you only need to put subdomain itself to the host (e.g., "blog" as in blog.example.com).

Besides, the value of CNAME usually ends with a dot.

Closes: 

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline (this link will be available after opening the PR).

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://docs.github.com/en/contributing/collaborating-on-github-docs/self-review-checklist).

### CODE REVIEW

**Comment by Lagosstate1268** [comment]
I need my account privacy because hackers keep getting my data

**Comment by nguyenalex836** [comment]
@theandyyeh Thanks so much for opening a PR! I'll get this triaged for review ✨

**Review by janbrasna — COMMENTED** [review]
In that case the terminal dot should be in the org example too for consistency:

File: content/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site.md [hunk 1]
```diff
@@ -163,7 +163,7 @@ To configure an apex domain, you only need to pick a single DNS record type from
 | Apex domain<br />(`example.com`) | `A` | `@` | `185.199.108.153`<br />`185.199.109.153`<br />`185.199.110.153`<br />`185.199.111.153` |
 | Apex domain<br />(`example.com`) | `AAAA` | `@` | `2606:50c0:8000::153`<br />`2606:50c0:8001::153`<br />`2606:50c0:8002::153`<br />`2606:50c0:8003::153` |
 | Apex domain<br />(`example.com`) | `ALIAS` or `ANAME` | `@` | `USERNAME.github.io` or<br /> `ORGANIZATION.github.io` |
```
[the remark this answers is not in the document]
Comment (janbrasna) [reply content/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site.md:165]: Similar table is used in https://github.com/github/docs/blob/main/content/pages/getting-started-with-github-pages/securing-your-github-pages-site-with-https.md#verifying-the-dns-configuration

**Review by nguyenalex836 — APPROVED** [review]

**Comment by nguyenalex836** [comment]
@theandyyeh Thanks so much for opening a PR with this fix! I'll update the branch and get this merged once tests are passing 🍏

**Pull request merged**